content/docs/examples/multicluster/index.md

```diff
@@ -0,0 +1,312 @@
+---
+title: Install Istio Multicluster on Google Kubernetes Engine clusters
+description: Example multicluster GKE install of Istio.
+weight: 65
+keywords: [kubernetes,multicluster]
+---
+
+This is an example deployment of multicluster Istio on 2
+[Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/) clusters.  It is an example
+using the [Istio kubernetes multicluster install instructions](/docs/setup/kubernetes/multicluster-install/).
+
+## Before you begin
+
+In addition to the prerequisites for installing Istio the following setup is required for this example:
+
+*  This sample requires a valid Google Cloud Platform project with billing enabled. If you are
```

Comment:
make this a block with `>`

Reply:
Well, I followed the pattern of the other content in the "Examples" navigation section.  They've used bullets in the "Before you begin" section.  I'd like to keep it consistent